exchanges/gateio/gateio.go

```diff
@@ -3378,8 +3378,8 @@ func (g *Gateio) CancelDeliveryPriceTriggeredOrder(ctx context.Context, settle,
 
 // ********************************** Options ***************************************************
 
-// GetAllUnderlyings retrieves all option underlyings
-func (g *Gateio) GetAllUnderlyings(ctx context.Context) ([]OptionUnderlying, error) {
+// GetAllOptionsUnderlyings retrieves all option underlyings
+func (g *Gateio) GetAllOptionsUnderlyings(ctx context.Context) ([]OptionUnderlying, error) {
 	var underlyings []OptionUnderlying
 	return underlyings, g.SendHTTPRequest(ctx, exchange.RestSpot, optionUnderlyings, &underlyings)
```

Comment:
Bump still not completed [HERE](https://github.com/thrasher-corp/gocryptotrader/pull/1058#discussion_r1022274635)